apps/app/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -355,7 +354,10 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_IDENTITY = "Apple Development";
```

Comment:
I configured my env to be able to test on my iPhone and it added this code, doesn't seem to negatively affect anything...